lib/internal/Magento/Framework/App/Response/HttpInterface.php

```diff
@@ -16,4 +16,49 @@
      * @return void
      */
     public function setHttpResponseCode($code);
+
```

Comment:
That is true. Given the youth of Magento 2 I think that case isn't very likely though. Plus, thanks to semver, backward incompatible changes are actually quite possible.